✨(app) create a redis-sentinel application (#402)
openfun/arnold: CHANGELOG.md

## [Unreleased]

### Added

- Create a new app redis-sentinel. This app uses the redis-operator 
  (https://github.com/spotahome/redis-operator) and it must be installed
  on your OpenShift cluster before deploying.

### Removed

> - probes on edxapp workers. Existing probes are not compatible with redis sentinel.

Reviewer: ```suggestion
- Probes on edxapp workers. Existing probes are not compatible with redis sentinel.
```

  To use them with redis sentinel, we must use a Django management command or Celery

>   using Django settings, doing this is too slow to be considered as a relevant alternative.

Reviewer: ```suggestion
  using Django settings, but doing so is too slow to be considered as a relevant alternative.
```

>   This removal is temporary, a new solution must be developed soon.

Reviewer: ```suggestion
  This removal is temporary, a new solution will be developed soon.
```

  (https://github.com/spotahome/redis-operator) and it must be installed
  on your OpenShift cluster before deploying.

### Removed

> - probes on edxapp workers. They are too slow to be run

Reviewer: This is not true: the real problem is that existing probes are not compatible with redis sentinel and alternative probes based on Django management commands or Celery using Django settings are too slow to be considered as a relevant alternative. You must add that this is a temporary removal until we implement a new efficient probe.
